content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -0,0 +1,270 @@
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Mask only input elements
+
+With the `mask user input` setting enabled, any input field is replaced with anonymized text. 
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+   {{< code-block lang="javascript" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+   // mask only input elements
+   val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
+       .setPrivacy(SessionReplayPrivacy.MASK_USER_INPUT)
+       .build()
+   SessionReplay.enable(sessionReplayConfig)
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+   {{< code-block lang="swift" filename="AppDelegate.swift" disable_copy="false" collapsible="true" >}}
+   // mask only input elements
+   let configuration = SessionReplayConfiguration(privacy: .maskUserInput)
+   sessionReplay = SessionReplay.initialize(with: configuration)
+   sessionReplay.start()
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Allow all (no masking)
+
+With the `allow all` setting enabled, all text is revealed.
+
+{{< img src="real_user_monitoring/session_replay/mobile/masking-mode-allow-all.png" alt="What your application screen may resemble when `allow all` is enabled." style="width:60%;">}}
+
+**Note**: Even with this option enabled, any sensitive text fields such as passwords, emails, phone numbers, and addresses are still masked. For more information, see [Text masking definitions](#text-masking-definitions).
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+   {{< code-block lang="javascript" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+   // no masking; all text is revealed
+   val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
+      .setPrivacy(SessionReplayPrivacy.ALLOW)
+      .build()
+   SessionReplay.enable(sessionReplayConfig)
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+   {{< code-block lang="swift" filename="AppDelegate.swift" disable_copy="false" collapsible="true" >}}
+
+   // no masking; all text is revealed
+   let configuration = SessionReplayConfiguration(privacy: .allowAll)
+   sessionReplay = SessionReplay.initialize(with: configuration)
+   sessionReplay.start()
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Unmask data in all recorded content
+By default, the Session Replay recorder masks all recorded content with `*` to ensure no sensitive information is visible in the recorded session. If you want to change this, add this option to **unmask data in all recorded content**:
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+   {{< code-block lang="kotlin" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+   val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
+    ...
+   .setSessionReplayPrivacy(SessionReplayPrivacy.[PRIVACY])
+   .build()
+
+   {{< /code-block >}}
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+This option is not available for iOS.
```

Comment:
@ncreated is this accurate, or is there a code snippet available?

Reply:
The entire `### Unmask data in all recorded content` section doesn't make sense IMO and we should consider removing it. "Unmasking" is achieved with `### Allow all (no masking)`, so there is nothing new we can write here.

I share @jamiemilstein thoughts from [above comment](https://github.com/DataDog/documentation/pull/18440/files#r1272655652).

Reply:
Removing this section @jamiemilstein